### ISSUE TITLE
No ui for Bigger Reactors Turbine or Condencer 

### ISSUE THREAD

**Issue by ptb666777** [AllTheMods/All-the-mods-7-Sky#254]
Probably spelled that wrong.
Reactor seems to work fine or as intended as documentation for that mod is scarce. However, the computer ports for the other two Multi-blocks do not show User Interface. I'm not sure if they are working as I can't see whats going on. multi-blocks do complete. It is entirely possible they are working and just don't have requirements met for operation. Updated from .6 to .9 before construction. Also went into creative to tinker and lose all ability to break multi-blocks and or interact with blocks near by.

**Comment by Mitchell5200** [comment]
For Bigger Reactors the computer ports do not have a GUI interface for them, and if the turbine and heat exchanger did form correctly, use the terminal blocks to interact with them, otherwise right click a casing or glass block to find out what's not letting them form

**Comment by ptb666777** [comment]
Yep, that was the issue. I thought the computers where the terminals. Had to have looked thru block list for mod 30 times and did not notice the terminal blocks. Issue resolved.
